php/php-k8s-v82/etc/nginx/ratelimit.conf

```diff
@@ -3,12 +3,15 @@ limit_req_status 429;
 
 ## Determine if this is a bot request via the user-agent string.
 map $http_user_agent $isbot_ua {
-        default                0;
-        ~*pingdom              0;
-        ~*(bot|crawler|spider) 1;
-        ~*facebookexternalhit  1;
-        ""                     1;
-        "-"                    1;
+        default                1;   # The new default is what we assume you are a bot!
+        ~*pingdom              0;   # Pingdom is a bot, but allowed.
+        ~*elb-healthchecker    0;   # The AWS health check is allowed.
+        ~*(bot|crawler|spider) 1;   # You are scrapers, thus bots.
+        ~*facebookexternalhit  1;   # Deffo a bot.
+        ~*0\.0\.0              1;   # Masking version? Bot.
+        ""                     1;   # Empty UA? Bot!
+        "-"                    1;   # No UA? probably a bot.
+        ~\/                    0;   # If your UA contains a slash, you may be human. This implies UAs *without* a slash are all bots.
```

Comment:
negative:
```
"~^[^/]*$"   1;  # no slash: probably not bot
```

Reply:
Well annoyingly that didn't work when I tied it yesterday, but now it does. Will update the PR (also means I should flip the default back to 0)

Reply:
as long as people are not considered guilty by default, I don't mind having the requests being assumed guilty by default :)